packages/realm-web/CHANGELOG.md

```diff
@@ -1,3 +1,19 @@
+?.?.? Release notes (2020-??-??)
+=============================================================
+
+### Breaking Changes
+* None
+
+### Enhancements
+* None
+
+### Fixed
+* Fixed failures while hydrating an app, such that  ([#3549](https://github.com/realm/realm-js/pull/3549))
```

Comment:
```suggestion
* A more robust implementation of app-hydration. Errors are now logged instead of thrown. ([#3549](https://github.com/realm/realm-js/pull/3549))
```